LICENSE

```diff
@@ -1,21 +1,202 @@
+      except as required for reasonable and customary use in describing the
+      origin of the Work and reproducing the content of the NOTICE file.
+
+   7. Disclaimer of Warranty. Unless required by applicable law or
+      agreed to in writing, Licensor provides the Work (and each
+      Contributor provides its Contributions) on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+      implied, including, without limitation, any warranties or conditions
+      of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+      PARTICULAR PURPOSE. You are solely responsible for determining the
+      appropriateness of using or redistributing the Work and assume any
+      risks associated with Your exercise of permissions under this License.
+
+   8. Limitation of Liability. In no event and under no legal theory,
+      whether in tort (including negligence), contract, or otherwise,
+      unless required by applicable law (such as deliberate and grossly
+      negligent acts) or agreed to in writing, shall any Contributor be
+      liable to You for damages, including any direct, indirect, special,
+      incidental, or consequential damages of any character arising as a
+      result of this License or out of the use or inability to use the
+      Work (including but not limited to damages for loss of goodwill,
+      work stoppage, computer failure or malfunction, or any and all
+      other commercial damages or losses), even if such Contributor
+      has been advised of the possibility of such damages.
+
+   9. Accepting Warranty or Additional Liability. While redistributing
+      the Work or Derivative Works thereof, You may choose to offer,
+      and charge a fee for, acceptance of support, warranty, indemnity,
+      or other liability obligations and/or rights consistent with this
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
sure, or Red Hat :)

On Wed, Jul 6, 2016 at 8:27 PM, Evan Shortiss <notifications@github.com>
wrote:

> In LICENSE
> <https://github.com/feedhenry-staff/fhlog/pull/1#discussion_r69784976>:
>
> > +      of your accepting any such warranty or additional liability.
> > +
> > +   END OF TERMS AND CONDITIONS
> > +
> > +   APPENDIX: How to apply the Apache License to your work.
> > +
> > +      To apply the Apache License to your work, attach the following
> > +      boilerplate notice, with the fields enclosed by brackets "[]"
> > +      replaced with your own identifying information. (Don't include
> > +      the brackets!)  The text should be enclosed in the appropriate
> > +      comment syntax for the file format. We also recommend that a
> > +      file or class name and description of purpose be included on the
> > +      same "printed page" as the copyright notice for easier
> > +      identification within third-party archives.
> > +
> > +   Copyright [yyyy] [name of copyright owner]
>
> Not sure about this, what do we put here? My name?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/feedhenry-staff/fhlog/pull/1/files/07e85a1355aa4e659e6ea4ea0d9cb0ea70803231#r69784976>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJnzgNWhBcs-mDjkJnzvZo1hYmW-Ex9ks5qS_N4gaJpZM4JGXHw>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
twitter: http://twitter.com/mwessendorf